src/libs/ajax.js

```diff
@@ -24,7 +24,17 @@ window.ajaxOverrideUtils = {
 
 const authOpts = (token = getUser().token) => ({ headers: { Authorization: `Bearer ${token}` } })
 const jsonBody = body => ({ body: JSON.stringify(body), headers: { 'Content-Type': 'application/json' } })
-const appIdentifier = { headers: { 'X-App-ID': 'Saturn' } }
+const appIdentifier = {
```

Comment:
TOL: This will work and presumably won't require any backend changes, but I think we should check in with whoever is consuming this data and make sure it won't make their analysis more difficult. In particular, queries looking for stuff from any version of the UI would need to be rewritten to include all these new tokens, and would be at risk for missing data in the future if we added one.

Another way would be to store the variant info separately, e.g. by writing out the `Origin` header to the data warehouse. That way, consumers could filter the data at whichever granularity they want. That might require backend changes, though.